### DISCUSSION TITLE
Disable ride breakdowns for specific rides

### DISCUSSION THREAD

**Discussion by astraylife1** [OpenRCT2/OpenRCT2#17366] [Feature requests]
A great feature would be the option to disable breakdowns for specific rides under the mechanics tab of that ride. Hacked rides and shoe-string rides wouldn’t have to be changed to crooked house or lift to function properly. This would enable them to retain normal ride stats. Also useful for coasters that are prone to crash after breakdowns.

**Comment by ocalhoun6** [comment]
Also great for rides that are being used as scenery, or for music only.

**Comment by 733737** [comment]
I also suggest that we remove Spiral Slide breakdowns, because why would it break down IRL? I don't know if the OpenRCT2 developers can be swayed

**Reply by ocalhoun6** [reply]
I guess 'safety cutout' makes *some* sense. Assuming there's some safety system that helps prevent having too many people on the slide at once, that system could malfunction and refuse to let anyone else slide.

Though ... looking at IRL spiral slides, I doubt they have such safety systems. Probably just the ride operator's job to watch and limit how many people can slide at once.

-------

One thing that might *actually* affect IRL spiral slides is "somebody spilled something (or puked) while going down the slide, and now it needs to be cleaned up before anyone else wants to slide on it".

**Reply by Derpercoaster** [reply]
Add the circus and space rings to that. They don't make sense in terms of breaking down. Sure, an actor in a circus might have an accident, and a space ring might lose a screw or something, but think of how rare that is compared to restraints getting stuck, or a lifthill chain jamming, or a vehicle wheel seizing up.

**Reply by ocalhoun6** [reply]
For the circus, *maybe*. (Though there are certainly things that could go wrong with the show, it seems like few of those things are something a mechanic could fix.)

For the space rings, though? Those things are fairly complicated mechanical contraptions, and they would definitely require maintenance from time to time. (And don't forget the DLC alternate skins of them, including a freaking teleportation machine!) For the absolute *best* realism, you might have only one ring (vehicle) break down, while leaving the other three operational, as would probably happen if a real life version had a problem with one of them. But that would require a lot of special code for very marginal benefit.

**Reply by astraylife1** [reply]
My main idea was just a checkbox so for individual rides you could choose whether to disable all breakdowns or not. Basically adding a new feature and not changing how the existing game works.
